core/modules/installer/installer.authorize.inc

```diff
@@ -201,37 +201,26 @@ function installer_authorize_update_batch_finished($success, $results) {
     // update data and recompute our status, so prevent show bogus results.
     _installer_authorize_clear_update_status();
 
-    // Take the site out of maintenance mode if it was previously that way.
-    if ($offline && isset($_SESSION['maintenance_mode']) && $_SESSION['maintenance_mode'] == FALSE) {
-      state_set('maintenance_mode', FALSE);
-      $page_message = array(
-        'message' => t('Update was completed successfully. Your site has been taken out of maintenance mode.'),
-        'type' => 'status',
-      );
-    }
-    else {
-      $page_message = array(
-        'message' => t('Update was completed successfully.'),
-        'type' => 'status',
-      );
-    }
+    $page_message = array(
+      'message' => t('Code replacement completed successfully.'),
```

Comment:
Can we please use something like `Files updated successfully.` here (same for the rest of the messages following)? The average Joe/Joette would not know the difference here, and "files" makes more sense than "code".

Reply:
The word `Files` has a different meaning than what we are talking about here -- it's the `files` directory, which is actually the opposite of what we are talking about here, which is code.  Any other recommendations?

Reply:
What about `Code replaced successfully`

Reply:
I've tried `Code updated...`